both/collections/roundProgress.js

```diff
@@ -0,0 +1,40 @@
+// Add functions to the Mongo object, using transform (see http://docs.meteor.com/#/full/mongo_collection)
+RoundProgress = function(doc) {
+  _.extend(this, doc);
+};
+
+_.extend(RoundProgress.prototype, {
```

Comment:
I'm not in love with the name. For one thing "progress" doesn't really have a plural form, so "progresses" is goofy and jarring.

But I can't think of an obviously better name...